app/controlplane/Makefile

```diff
@@ -33,10 +33,15 @@ migration_apply: check-atlas-tool migration_hash
 	atlas migrate status --dir ${local_migrations_dir} --url ${local_db}
 	atlas migrate apply --dir ${local_migrations_dir} --url ${local_db}
 
+.PHONY: migration_update
```

Comment:
this is just some refactoring to add a new make target for creating empty migration files. Before, `migration_new` was not actually creating empty migrations but making sure the migrations were in sync.